tests/datasource/fluent/integration/test_sql_datasources.py

```diff
@@ -753,7 +753,6 @@ class TestColumnIdentifiers:
     @pytest.mark.parametrize(
         "expectation_type",
         [
-            "expect_column_to_exist",
```

Comment:
Why are we removing this expectation type?

Reply:
`mostly` is not valid for this Expectation